src/ansys/fluent/core/solver/flobject.py

```diff
@@ -1585,6 +1602,12 @@ def execute_command(self, *args, **kwds):
                 assert_type(ret, base_t._state_type)
             return ret
 
+    def _interrupt(self):
+        try:
+            self.root.solution.run_calculation.interrupt()
+        except AttributeError:
+            raise KeyboardInterrupt
```

Comment:
If I'm understanding this correctly, raising `KeyboardInterrupt` here doesn't seem appropriate. It should raise a more appropriate exception/error, and propagate whatever message was in the `AttributeError` that was caught